### ISSUE TITLE
Ensure consistent tabindex for fields

### ISSUE THREAD

**Issue by hzbarcea** [edgeid/auth41-plugins#11]
plugins/auth41-federation-broker/src/main/resources/theme/base/login/account-chooser.ftl

There's a skip in tabindex values from 1 (input field on line 12) to 4 (submit button). This creates a non-sequential tab order which can be confusing for keyboard users. The tabindex should be sequential (1, 2, 3...) or omitted entirely to use the natural DOM order. The link on line 34 uses tabindex="5" which would also need adjustment.
